src/hb-set.hh

```diff
@@ -53,8 +53,8 @@ struct hb_set_t
 
   struct page_t
   {
-    inline void init0 (void) { memset (reinterpret_cast<char*>(&v), 0, sizeof (v)); }
-    inline void init1 (void) { memset (reinterpret_cast<char*>(&v), 0xff, sizeof (v)); }
+    inline void init0 (void) { memset ((void *)(&v), 0, sizeof (v)); }
+    inline void init1 (void) { memset ((void *)(&v), 0xff, sizeof (v)); }
```

Comment:
HarfBuzz code style apparently prefers space after casting operator, just saying for future works :)